packages/fx-core/templates/plugins/resource/bot/bicep/bot.bicep

```diff
@@ -0,0 +1,97 @@
+param botServiceName string
+param botServerfarmsName string
+param botWebAppSKU string = 'F1'
+param botServiceSKU string = 'S1'
```

Comment:
is there free tier? if yes, we should always use free tier

Reply:
Fixed.